cmd/loader.go

```diff
@@ -174,7 +178,7 @@ func runTraceMode(cfg *config.LoaderConfiguration, readIATFromFile bool, justGen
 	yamlPath := parseYAMLSpecification(cfg)
 
 	traceParser := trace.NewAzureParser(cfg.TracePath, durationToParse)
-	functions := traceParser.Parse(cfg.Platform)
+	functions := traceParser.Parse(cfg.Platform)		// Indexed by hashfunction names
```

Comment:
I don't see any indexing here. What does this comment means then?

Reply:
This was more for my understanding when modifying the code, that the hashfunction is what I would use to address the function that the specific hash function is mapped to. Will remove it.

Reply:
You don't use it for indexing. `functions` is a flat array.